test_conformance/math_brute_force/main.c

```diff
@@ -110,6 +110,7 @@ cl_device_fp_config gHalfCapabilities = 0;
 int             gWimpyReductionFactor = 32;
 int             gWimpyBufferSize = BUFFER_SIZE;
 int             gVerboseBruteForce = 0;
+cl_device_type gDeviceType = CL_DEVICE_TYPE_DEFAULT;
```

Comment:
This seems unrelated to fp16 support. Could you split this out, please?

Also, shouldn't this be in the `#if defined( __APPLE__)` section below?